.github/workflows/report-size.yml

```diff
@@ -0,0 +1,103 @@
+name: Report size
+
+on:
+  workflow_run:
+    workflows: ["Read size"]
```

Comment:
On this PR the "read size" workflow has completed but not "report size". Is it a bug?

Reply:
No, it is correct behaviour. 

The `workflow_run` event is run in the context of the base branch (`dev`) to be able to have write access to the pull request.
However the action is not yet on `dev`, meaning it won't run until this PR has been merged.